awx/main/models/jobs.py

```diff
@@ -1140,6 +1140,8 @@ class SystemJobOptions(BaseModel):
         ('cleanup_jobs', _('Remove jobs older than a certain number of days')),
         ('cleanup_activitystream', _('Remove activity stream entries older than a certain number of days')),
         ('cleanup_facts', _('Purge and/or reduce the granularity of system tracking data')),
```

Comment:
```diff
-
```

Reply:
This is what was causing Django to add that choice in the migration.  I will remove it.